Examples/Io/Csv/src/GeometryExampleBase.cpp

```diff
@@ -0,0 +1,176 @@
+// This file is part of the Acts project.
```

Comment:
I think I am missing something about this file. Why is this a copy of GeometryExampleBase inside  Examples/Io/Csv ?

Reply:
Ha, that's a mistake. Weird.

Reply:
Deleted it.